variables.tf

```diff
@@ -55,14 +55,14 @@ variable "deny" {
 }
 
 variable "exclude_folders" {
-  description = "List of folders to exclude from the policy"
-  type        = list(string)
+  description = "Set of folders to exclude from the policy"
+  type        = set(string)
   default     = [""]
 }
 
 variable "exclude_projects" {
-  description = "List of projects to exclude from the policy"
-  type        = list(string)
+  description = "Set of projects to exclude from the policy"
+  type        = set(string)
   default     = [""]
```

Comment:
for_each will detect this default [""] as 1 element and will proceed  to create` resource "google_project_organization_policy" "project_policy_list_exclude_projects"` with `""` as project which will fail when we are calling only `exclude_folders`.[ Example invocation where failure happens]( https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/examples/boolean_org_exclude/main.tf)

Reply:
Thanks. I would have expected this to already fail before, though

Reply:
It was not previously failing due to the way we were calculating length
`length(compact(var.exclude_folders))`. Compact got rid of those empty `""` which then gave a length of zero